src/tendencies_parametrizations.jl

```diff
@@ -8,20 +8,23 @@ Extract for each vertical atmospheric column the prognostic variables
 grid-points, compute all parametrizations on a single-column basis,
 then write the tendencies back into a horizontal field of tendencies.
 """
-function parametrization_tendencies!(   diagn::DiagnosticVariables{NF},
-                                        M::PrimitiveEquationModel,
-                                        ) where NF
-
+function parametrization_tendencies!(
+    diagn::DiagnosticVariables{NF},
+    M::PrimitiveEquationModel,
+) where {NF}
```

Comment:
Hilarious how we have the constant battle how to indent the function arguments! Just to clarify, I don't mind either way 😉

Reply:
Haha blame [JuliaFormatter.jl](https://github.com/domluna/JuliaFormatter.jl), not me ;)

Reply:
Once you've touched every file we'll have a consistent style! YAY!